README.md

```diff
@@ -23,34 +23,35 @@ Scripts for updating and debugging Kafka can be found [here](https://github.com/
-| FILTER_API_AUTH_TOKEN       | FD0108EA-825D-411C-9B1D-41EF7727F465 | The auth token for the filter API
-| AWS_REGION                  | eu-west-1                            | The AWS region to use
-| S3_BUCKET_URL               | _unset_     (e.g. `https://cf.host`) | If set, the URL prefix for public, exported downloads
-| S3_BUCKET_NAME              | csv-exported                         | The name of the public S3 bucket to store exported files
-| S3_PRIVATE_BUCKET_NAME      | csv-exported                         | The name of the private s3 bucket to store exported files
-| CSV_EXPORTED_PRODUCER_TOPIC | common-output-created                | The topic to add messages to when a job is complete
-| ERROR_PRODUCER_TOPIC        | filter-error                         | The topic to add messages to when an error occurs
-| HEALTHCHECK_INTERVAL        | time.Minute                          | How often to run a health check
-| GRACEFUL_SHUTDOWN_TIMEOUT   | time.Second * 10                     | How long to wait for the service to shutdown gracefully
-| DOWNLOAD_SERVICE_URL        | http://localhost:23600               | The URL of the download service
-| VAULT_ADDR                  | http://localhost:8200                | The address of vault
-| VAULT_TOKEN                 | -                                    | Use `make debug` to set a vault token
-| VAULT_PATH                  | secret/shared/psk                    | The vault path to store psks
-| SERVICE_AUTH_TOKEN          | 0f49d57b-c551-4d33-af1e-a442801dd851 | The service token for this app
-| ZEBEDEE_URL                 | http://localhost:8082                | The URL to zebedee
-| AWS_ACCESS_KEY_ID           | -                                    | The AWS access key credential
-| AWS_SECRET_ACCESS_KEY       | -                                    | The AWS secret key credential
-| FULL_DATASET_FILE_PREFIX    | full-datasets                        | The prefix added to full dataset download files
-| FILTERED_DATASET_FILE_PREFIX| filtered-dataset                     | The prefix added to filtered dataset download files
+| Environment variable         | Default (example)                    | Description
+| ---------------------------- | ------------------------------------ | -----------
+| BIND_ADDR                    | :22500                               | The host and port to bind to
+| KAFKA_ADDR                   | localhost:9092                       | The address of Kafka
+| FILTER_JOB_CONSUMER_TOPIC    | filter-job-submitted                 | The name of the topic to consume messages from
+| FILTER_JOB_CONSUMER_GROUP    | dp-dataset-exporter                  | The consumer group this application to consume filter job messages
+| DATASET_API_URL              | http://localhost:22000               | The URL of the dataset API
+| DATASET_API_AUTH_TOKEN       | FD0108EA-825D-411C-9B1D-41EF7727F465 | The auth token for the dataset API
```

Comment:
I think these tokens can be removed from configuration and README.md:

DATASET_API_AUTH_TOKEN
FILTER_API_AUTH_TOKEN

as the application should be using it's SERVICE_AUTH_TOKEN to authenticate against other services

This will probably be best as a clean up task in another story